### ISSUE TITLE
Add tags to the simple theme

### ISSUE THREAD

**Issue by gsar-mdsol** [gkushang/cucumber-html-reporter#150]
Is it possible to add the tags to the simple theme? I am happy to implement it.

**Comment by gsar-mdsol** [comment]
Is this project still maintained?

**Comment by znevrly** [comment]
I hope so, I want to add some issues and pull requests soon :)

**Comment by gkushang** [comment]
@gsar-mdsol 

Yes, this project is still maintained. Please go ahead and create a PR. I'd appreciate your contributions.
